packages/hadron-build/commands/release.js

```diff
@@ -321,10 +321,6 @@ const installDependencies = util.callbackify(async(CONFIG) => {
  * @api public
  */
 const removeDevelopmentFiles = (CONFIG, done) => {
-  if (CONFIG.platform !== 'darwin') {
-    done();
-    return;
-  }
```

Comment:
Oh wow

Reply:
Looks like this change took 5+MB off of non-mac releases 🔥